Remove my BSQ explorer

devinbileck opened the pull request:
I can retire my node now that @runbtc added his BSQ explorer in #7422.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Chores**
  - Removed "mempool.bisq.services" from the list of available blockchain explorers and marked it as deprecated.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

alejandrogarcia83 commented:
I can't merge this PR because due to a merge conflict. Could you please rebase your PR to master? Thanks!

devinbileck committed:
Remove my BSQ explorer

I can retire my node now that @runbtc added his BSQ explorer
in bisq-network#7422.

devinbileck commented:
@alejandrogarcia83 should be good now

alejandrogarcia83 reviewed (approved):
utACK
